Sources/KSCrashRecording/include/KSCrashReportFilter.h

```diff
@@ -37,7 +39,7 @@ NS_ASSUME_NONNULL_BEGIN
  *                  user cancelling the operation).
  * @param error Non-nil if an error occurred.
  */
-typedef void(^KSCrashReportFilterCompletion)(NSArray<NSDictionary<NSString *,id> *>* _Nullable filteredReports,
+typedef void(^KSCrashReportFilterCompletion)(NSArray<KSCrashReport*>* _Nullable filteredReports,
```

Comment:
The key issue was here. This callback would crash if implemented with Swift and the value type is not a dictionary.